app-emulation/uade/uade-2.13-r1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Unix Amiga Delitracker Emulator - plays old Amiga tunes through UAE emulation"
+HOMEPAGE="https://zakalwe.fi/uade"
+SRC_URI="https://zakalwe.fi/uade/uade2/${P}.tar.bz2"
+
+LICENSE="GPL-2+ LGPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+RDEPEND="media-libs/libao"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+DOCS=( AUTHORS ChangeLog doc/BUGS doc/PLANS )
+
+PATCHES=( "${FILESDIR}"/${P}-configure.patch )
+
+src_configure() {
+	tc-export CC
+
+	./configure \
+		--prefix=/usr \
```

Comment:
I just noticed that this isn't respecting prefix. Please change this to `--prefix="${EPREFIX}"/usr`.